### DISCUSSION TITLE
"Lastest Version Download" or Server files Navigation

### DISCUSSION THREAD

**Discussion by glowingstone124** [PurpurMC/Purpur#1370] [__Dismissed__] [answered]
Will you be adding a "Latest Download" feature in the future development? I would like to regularly wget the latest Purpur server files, but I need to manually download them every time. Is there a way to automate the downloading of the latest files?

**Comment by Roan-V** [comment, marked as answer]
That already exists through the API
https://api.purpurmc.org for the syntax
here is the latest build atm:
https://api.purpurmc.org/v2/purpur/1.20.1/latest/download

The only thing that doesn't automatically grab latest is the minecraft version, that you have to do manually
But that is a good thing, you don't want to suddenly update a major game version
